moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -127,12 +127,6 @@ ServoCalcs::ServoCalcs(const rclcpp::Node::SharedPtr& node,
         return changeControlDimensions(req, res);
       });
 
-  // ROS Server to reset the status, e.g. so the arm can move again after a collision
-  reset_servo_status_ = node_->create_service<std_srvs::srv::Empty>(
```

Comment:
Thanks for noticing that this isn't used! Please delete `reset_servo_status_` from the header file, too